src/Apps/CollectorProfile/CollectorProfileApp.tsx

```diff
@@ -0,0 +1,58 @@
+import { Text } from "@artsy/palette"
+import { MetaTags } from "Components/MetaTags"
+import { RouteTab, RouteTabs } from "Components/RouteTabs"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useFeatureFlag } from "System/useFeatureFlag"
+import { CollectorProfileApp_me$data } from "__generated__/CollectorProfileApp_me.graphql"
+
+const TABS = [
+  { name: "My Collection", url: "/collector-profile/collection" },
+  { name: "Insights", url: "/collector-profile/insights" },
+  { name: "Saves & Follows", url: "/collector-profile/saves" },
+]
+
+interface CollectorProfileAppProps {
+  me: CollectorProfileApp_me$data
+}
+
+const CollectorProfileApp: React.FC<CollectorProfileAppProps> = ({
+  me,
+  children,
+}) => {
+  const isCollectorProfileEnabled = useFeatureFlag("cx-collector-profile")
+
+  if (!isCollectorProfileEnabled) {
+    return null
+  }
```

Comment:
We need something like this 😂

![Transport-Under-Construction-icon](https://user-images.githubusercontent.com/11945712/204568212-498cc7c5-03cf-4f51-b048-d12340c5c965.png)